lib/ronin/cli/commands/archive.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+#
+# Copyright (c) 2006-2023 Hal Brodigan (postmodern.mod3 at gmail.com)
+#
+# Ronin is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ronin is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ronin.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/cli/file_processor_command'
+require 'ronin/support/archive'
+
+module Ronin
+  class CLI
+    module Commands
+      #
+      # Archive the files.
+      #
+      # ## Usage
+      #
+      #     ronin archive [option] [FILE ...]
+      #
+      # ## Options
+      #
+      #     -f, --format tar|zip             Archive format
+      #     -o, --output PATH                Archived file path
+      #
+      # ## Arguments
+      #
+      #     [FILE ...]                       Optional file(s) to archive
```

Comment:
Remove the `[` `]` from the `[FILE ...]` because at least one `FILE` must be specified.